### PR TITLE
Add docs about user management in AWS

### PR DESCRIPTION
**Pull request by surminus**
Add initial docs about user management in AWS.

This has two links missing:

 - official guidance on the central GDS users account, which should be provided by Reliability Engineering
 - details on GOV.UK accounts and IDs. These could potentially be stored in the password store.

https://trello.com/c/hH4jI14U/873-add-docs-on-user-management

### CODE REVIEW

File: source/manual/user-management-in-aws.html.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+If successful, this will output some credentials. Store them in your environment using
+these environment variables:
+
+```
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_SESSION_TOKEN
+```
```
Comment (afda16) [inline source/manual/user-management-in-aws.html.md]: Probably we could add how to switch role in the web console too

**Comment by afda16** [comment]
Should the document be in a new AWS section of the manual?

**Comment by surminus** [comment]
I don't think so. The header itself sets the section it's part of.

File: source/manual/user-management-in-aws.html.md (hunk 1 above)
Comment (surminus) [reply source/manual/user-management-in-aws.html.md]: I will expand this along with the linked documentation.

**Pull request merged**